internal/lib/restore.go

```diff
@@ -0,0 +1,153 @@
+package lib
+
+import (
+	"context"
+	"fmt"
+	"os"
+	"path/filepath"
+
+	metadata "github.com/checkpoint-restore/checkpointctl/lib"
+	"github.com/checkpoint-restore/go-criu/v5/stats"
+	"github.com/containers/podman/v4/pkg/checkpoint/crutils"
+	"github.com/cri-o/cri-o/internal/oci"
+	"github.com/opencontainers/runtime-tools/generate"
+	"github.com/sirupsen/logrus"
+)
+
+func (c *ContainerServer) restoreFileSystemChanges(ctr *oci.Container, mountPoint string) error {
```

Comment:
nit: can this function be defined below the function where it's used

Reply:
Sure. That is just what I am used to do in C :wink: